packages/form-system/stories/index.stories.mdx

```diff
@@ -0,0 +1,237 @@
+| [field](../../field)                   | Base class for all inputs          |
+| [form](../../form)                     | Wrapper for multiple form elements |
+| [input](../../input)                   | Input element for strings          |
+| [input-amount](../../input-amount)     | Input element for amounts          |
+| [input-date](../../input-date)         | Input element for dates            |
+| [input-email](../../input-email)       | Input element for e-mails          |
+| [input-iban](../../input-iban)         | Input element for IBANs            |
+| [radio](../../radio)                   | Radio form element                 |
+| [radio-group](../../radio-group)       | Group of radios                    |
+| [select](../../select)                 | Simple native dropdown element     |
+| [textarea](../../textarea)             | Multiline text input               |
+| [validate](../../validate)             | Validation for our form components |
+
+## Meta Package
+
+This is a meta package to show interaction between various form elements.
+For usage and installation please see the appropriate packages.
+
+## Umbrella Form
+
+<Story name="Umbrella Form">
+  {html`
+    <lion-form>
+      <form>
+        <lion-input
+          name="first_name"
+          label="First Name"
+          .validators="${[new Required()]}"
+        ></lion-input>
+        <lion-input
+          name="last_name"
+          label="Last Name"
+          .validators="${[new Required()]}"
+        ></lion-input>
+        <!-- TODO: lion-input-birthdate -->
+        <lion-input-date
+          name="date"
+          label="Date of application"
+          .modelValue="${'2000-12-12'}"
+          .validators="${[new Required()]}"
+        ></lion-input-date>
+        <lion-textarea
+          name="bio"
+          label="Biography"
+          .validators="${[new Required(), new MinLength(10)]}"
+          help-text="Please enter at least 10 characters"
+        ></lion-textarea>
+        <lion-input-amount name="money" label="Money"></lion-input-amount>
+        <lion-input-iban name="iban" label="Iban"></lion-input-iban>
+        <lion-input-email name="email" label="Email"></lion-input-email>
+        <lion-checkbox-group
+          label="What do you like?"
+          name="checkers"
+          .validators="${[new Required()]}"
+        >
+          <lion-checkbox name="checkers[]" value="foo" label="I like foo"></lion-checkbox>
+          <lion-checkbox name="checkers[]" value="bar" label="I like bar"></lion-checkbox>
+          <lion-checkbox name="checkers[]" value="baz" label="I like baz"></lion-checkbox>
+        </lion-checkbox-group>
+        <lion-radio-group
+          class="vertical"
+          name="dinosaurs"
+          label="Favorite dinosaur"
+          .validators="${[new Required()]}"
+          error-message="Dinosaurs error message"
+        >
+          <lion-radio name="dinosaurs[]" value="allosaurus" label="allosaurus"></lion-radio>
```

Comment:
These example was updated in the latest master:
https://github.com/ing-bank/lion/pull/449/

Please update here as well